pkg/notes/notes_data.go

```diff
@@ -513,3 +532,74 @@ func setEmptyComponentGitHubRepositoryFromOtherKilnfile(k1, k2 cargo.Kilnfile) c
 }
 
 func closeAndIgnoreError(c io.Closer) { _ = c.Close() }
+
+type TrainstatFetcher interface {
```

Comment:
I'd call this "TrainstatNotesFetcher" to make it align with the method name better.

Reply:
Yes, that makes sense. Done!